modin/backends/pandas/query_compiler.py

```diff
@@ -1589,6 +1592,98 @@ def clip(self, lower, upper, **kwargs):
             new_modin_frame = self._modin_frame._map(lambda df: df.clip(**kwargs))
         return self.__constructor__(new_modin_frame)
 
+    def corr(self, method="pearson", min_periods=1):
+        if method == "pearson":
+            numeric_self = self.drop(
+                columns=[
+                    i for i in self.dtypes.index if not is_numeric_dtype(self.dtypes[i])
+                ]
+            )
+            return numeric_self._nancorr(min_periods=min_periods, cov=False)
```

Comment:
`cov=False` can be excluded from this call

Reply:
Yes, I know, but I am keeping passing `cov` to the function for clarity.

Reply:
Then does the default value make sense?

Reply:
It is done for pandas API consistency. If passing `cov` to the function causes misunderstanding here, I can delete it.

Reply:
Yes, I think this should be deleted.

Reply:
my suggestion will be to rename the parameter `cov -> method` at a `_nancorr` method for more clarity. So `method` will be a string parameter, that can be "cov" or "corr".
What do you think?

Reply:
I think it would be redundant. Firstly, `corr` takes parameter `method` itself. If we would pass another parameter named `method` to the `_nancorr`, it would be confused per my opinion. Secondly, I would like to keep an interface of the method the same like in pandas because it would be easier to understand the logic for people, which already saw this function.